config/bundles.php

```diff
@@ -1,24 +1,5 @@
 <?php
 
-/*
```

Comment:
I know, it's boring cause symfony regenerates this file,
But header license should be restored